### DISCUSSION TITLE
empty value

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1088] [Q&A] [answered]
Hello!
In my process, three Get Text blocks look for a value on the page and then display them in an Alert.
![image](https://user-images.githubusercontent.com/120256667/211998751-fa5b2481-747e-483d-a8de-070e7d5ab568.png)

Sometimes there is no single value.
![image](https://user-images.githubusercontent.com/120256667/211998849-e3421fcd-34c0-4442-9370-b0a253e691c2.png)

What value will automaRefData('variables', 'phone1') have if the selector doesn't find the element?

**Comment by Kholid060** [comment, marked as answer]
It will return `undefined`

**Reply by Art-Che** [reply]
thanks It works!
